File: 2.cs
using System;

class Program {
    static void Main() {
        string sentence = Console.ReadLine(); // ввод строки
        int k = 0;
        for (char letter = (char)('a'); letter <= (char)('z'); letter++) { // перебор строки
            if (sentence.Contains(letter)) { // поиск буквы в строке
                k++; // подсчёт количества
            }
        }
    }
}

File: 2.kt
import java.util.Scanner;

fun main() {
    val scanner = Scanner(System.`in`)
    val sentence = scanner.nextLine() // ввод строки
    var k = 0
    for (letter in 'a'..'z') { // перебор алфавита
        if (sentence.contains(Character.toString(letter))) { // поиск букв в строке
            k++ // подсчёт количества
        }
    }
    if (k == 26) {
        print("true")
    }
    else {
        print("false")
    }
}